src/app/components/transaction/transaction-title.tsx

```diff
@@ -22,10 +22,11 @@ export function TransactionTitle(props: TransactionTitleProps) {
   return (
     <BasicTooltip disabled={!isEllipsisActive} label={title} side="top">
       <Title
-        fontWeight="normal"
+        fontWeight={500}
```

Comment:
We seem to use this a lot `fontWeight={500}`. Maybe we should have tokens for weight?